app/controllers/cloud_subnet_controller.rb

```diff
@@ -76,42 +76,45 @@ def create
     case params[:button]
     when "cancel"
       javascript_redirect :action    => 'show_list',
-                          :flash_msg => _("Add of new Subnet was cancelled by the user") % {:model => ui_lookup(:table => 'cloud_subnet')}
+                          :flash_msg => _("Add of new Subnet was cancelled by the user") % {
+                            :model => ui_lookup(:table => 'cloud_subnet')}
```

Comment:
The `:model =>` expansion here can safely go away, right? (Since there's no `%{model}` to expand really)
Not introduced by this PR, but can go away regardless.